docs/PYPI.md

```diff
@@ -1,190 +0,0 @@
-- logging
-- Http Client
-- Events
-- Event middleware
-- Commands
-- Command arguments *(for types: str, int, float, bool, User, Channel, Role)*
-- Command argument choices
-- Command argument descriptions
-- Command cool downs (Using WindowSliding technique)
-- Tasks
-- Cogs
-
-**Client base class example:**
-
-```py
-from pincer.client import Bot
-
-# Note that both `Bot` and `Client` are valid!
-bot = Bot("...")
-bot.run()
-```
-
-**An example on the `on_ready` event**
-
-```py
-from time import perf_counter
-from pincer.client import Client
-
-client = Client("...")
-
-
-@client.event
-async def on_ready():
-    print(f"Logged in as {client.bot} after {perf_counter()} seconds")
-
-
-client.run()
-```
-
-### Inherited client
-
-You have the possibility to use your own class to inherit from the Pincer bot
-base.
-
-```py
-from pincer import Client
-from pincer.commands import command, CommandArg, Description
-
-class Bot(Client):
-    def __init__(self) -> None:
-        super(Bot, self).__init__(token="...")
-
-    @Client.event
-    async def on_ready(self) -> None:
-        ...
-
-    @command(description="Say something as the bot!")
-    async def say(self, message: str):
-        return message
-
-    @command(description="Add two numbers!")
-    async def add(
-            self,
-            first: CommandArg[int, Description["The first number"]],
-            second: CommandArg[int, Description["The second number"]]
-    ):
-        return f"The addition of `{first}` and `{second}` is `{first + second}`"
-```
-
-For more examples you can take a look at the examples folder or check out our
-bot on GitHub:
-
-> <https://github.com/Pincer-org/Pincer-bot>
-
-### Advanced Usage
-
-#### Enable the debug mode
-
-_If you want to see everything that is happening under the hood, either out of
-curiosity or to get a deeper insight into the implementation of some features,
-we provide debug logging!_
-
-```py
-import logging
-
-logging.basicConfig(level=logging.DEBUG)
-```
-
-**Note:** _A lot of printing can happen, including sensitive information, so
-make sure to be aware of what you're doing if you're enabling it!_
-
-#### Middleware
-
-_From version 0.4.0-dev, the middleware system has been introduced. This system
-gives you the full freedom to remove the already existing middleware which has
-been created by the developers and create custom events. Your custom middleware
-directly receives the payload from Discord. You can't really do anything wrong
-without accessing the `override` attribute, but if you access this attribute the
-Pincer team will not provide any support for weird behavior. So in short, only
-use this if you know what you're doing. An example of using this with a custom
-`on_ready` event can be found
-[in our docs](https://pincer.readthedocs.io/en/latest/pincer.html#pincer.client.middleware)
-._
-
-## 🏷️ License
-
-`© 2021 copyright Pincer`
-
-This repository is licensed under the MIT License.
-
-See LICENSE for details.
```

Comment:
I'd prefer to stay with a separated README for PyPI as he dont support table display or banner from the repo README

Reply:
I reverted the change.